### DISCUSSION TITLE
Disable rate limiting in ApiDefinition

### DISCUSSION THREAD

**Discussion by MCBrandenburg** [TykTechnologies/tyk-operator#438] [General] [answered]
When looking to disable rate limiting for an API, I discovered I'm not able to do that via the `ApiDefinition` due to the code being commented out in [api.go](https://github.com/TykTechnologies/tyk-operator/blob/master/api/model/api.go#L681). I was wondering if there was a reason for having this functionality commented out

**Comment by carolwkc** [comment]
Hi @MCBrandenburg and thank you for raising this. This has been commented out probably just because we haven't properly test and document it yet. Please let us know if you want to raise the priority for this request.

**Comment by asoorm** [comment, marked as answer]
Just a quick update - this has now been addressed in v0.10.0 https://github.com/TykTechnologies/tyk-operator/releases/tag/v0.10.0
